docs/concepts.md

```diff
@@ -62,7 +62,10 @@ implemented via **channels**. A channel represents a uni-directional stream of
 messages, with a receive half and a send half that an Oak Node can read from or
 write to respectively. Each half of a channel is identified by a **handle**,
 which is used as a parameter to the corresponding
-[host function](abi.md#host-functions) calls.
+[host function](abi.md#host-functions) calls. Channel handles are integer values
+that are specific to a particular Node (like per-process UNIX file descriptors),
+so handle value 3 for Node A identifies a different channel than handle value 3
```

Comment:
```suggestion
so handle value 3 for Node A may identify a different channel than handle value 3
```

Reply:
I added " (unless there happens to be a coincidence of numbering)" at the end of the sentence, to keep the strength of the original wording (as there's only a 1-in-2^64 chance of getting the same values for a shared channel).